### PR TITLE
Performance: Standardize lodash imports to use destructured members

### PR DESCRIPTION
**Pull request by kaydelaney**
**What this PR does / why we need it**:
Previously our bundles had lodash imports of the form `import x from 'lodash/x'` and `import { x } from 'lodash'`, as well as `import _ from 'lodash'`. Since the entire lodash package needs to be part of the bundle for plugin support, using method imports just increases the overall bundle size.
This PR changes all method imports to destructured member imports in order to reduce overall bundle size.

Old:
Screen Shot 2021-04-15 at 17 40 30

New:
Screen Shot 2021-04-15 at 17 40 44

Relates #30483

### CODE REVIEW

**Comment by leeoniya** [comment]
naturally, i find this to be fantastic :tada:

if you have time in the future to make Slate & plugins lazy-loaded, it would have a much larger impact, too ;)

File: packages/grafana-ui/src/components/Graph/Graph.tsx [hunk 1]
```diff
@@ -1,10 +1,9 @@
 // Libraries
 import $ from 'jquery';
 import React, { PureComponent } from 'react';
-import uniqBy from 'lodash/uniqBy';
+import _, { uniqBy } from 'lodash';
```
Comment (joshhunt) [inline packages/grafana-ui/src/components/Graph/Graph.tsx:4]: I'm not sure why it's not a lint error, but I don't think the default `_` export is even used in here?

**Comment by ifrost** [comment]
Nice! 🚀  Could we also add an eslint rule to enforce it in the future? https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/import-scope.md

**Comment by kaydelaney** [comment]
> Nice! 🚀 Could we also add an eslint rule to enforce it in the future? https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/import-scope.md

I'd considered this, but as far as I can tell there's no way to allow _both_ a full import (`import _ from 'lodash'`) and member imports (`import { x } from 'lodash'`) using that plugin. Maybe that's not a big deal though?

**Review by ryantxu — APPROVED** [review]
> no way to allow both a full import

I think adding `// eslint-disable-next-line` comment for the rare cases when we really want `_` is fine.

**Pull request merged**